conf/awsecr.config

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Be nice to stick to the same comment styles as we have in other config files in the template:
https://github.com/nf-core/tools/blob/2dc8d24df46c33eef69dfe0cd4e31916452a4126/nf_core/pipeline-template/conf/base.config#L1-L9